test/src/main/java/org/apache/accumulo/test/functional/MemoryStarvedScanIT.java

```diff
@@ -307,13 +308,16 @@ public void testBatchScanReturnsEarlyDueToLowMemory() throws Exception {
         consumeServerMemory(scanner);
 
         // Wait for longer than the memory check interval
-        Thread.sleep(6000);
+        Thread.sleep(3000);
```

Comment:
Why do you still have a sleep here? The main point of using the Wait.waitFor() api is you don't need to include anymore artificial sleeps as it will progress when ready.

Reply:
The metrics collection is inherently asynchronous - this was to insert some delay and then use the polling in Wait so that the back off could start small and then start backing off, hopefully reducing the over-all time required for the test by minimizing the total time spent waiting for the metrics to update.

This test in particular seems to have a race condition with the low memory detector that was recently added - so it was the only one addressed here.  The others were left alone for now